frame/im-online/README.md

```diff
@@ -10,7 +10,7 @@ in the current era or session.
 
 The heartbeat is a signed transaction, which was signed using the session key
 and includes the recent best block number of the local validators chain as well
-as the [NetworkState](https://docs.rs/../client/offchain/struct.NetworkState.html).
+as the [NetworkState](https://docs.rs/../latest/../client/offchain/struct.NetworkState.html).
```

Comment:
Ok, this one is wrong. Not sure if the related package is published though. What I could do is remove the link completely if the URI starts with `../../` (for now).